src/SceneKit/SCNCompat.cs

```diff
@@ -52,16 +53,32 @@ public virtual void SetTimingFunction (Action<float> timingFunction)
 	}
 #if TVOS && !XAMCORE_4_0
 	partial class SCNMaterialProperty {
+#if !NET
 	[iOS (8, 0)]
 		[Deprecated (PlatformName.iOS, 10, 0)]
 		[Deprecated (PlatformName.TvOS, 10, 0, message: "This API has been totally removed on tvOS.")]
+#else
+		[UnsupportedOSPlatform ("ios10.10")]
+		[UnsupportedOSPlatform ("tvos10.10")]
+#if TVOS
+		[Obsolete ("Starting with tvos10.10 this API has been totally removed on tvOS.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
+#endif
 		public virtual NSObject? BorderColor { get; set; }
 	}
 
 	partial class SCNRenderer {
+#if !NET
 		[iOS (8, 0)]
 		[Deprecated (PlatformName.iOS, 9, 0)]
 		[Deprecated (PlatformName.TvOS, 10, 0, message: "This API has been totally removed on tvOS.")]
+#else
+		[UnsupportedOSPlatform ("ios10.10")]
+		[UnsupportedOSPlatform ("tvos10.10")]
+#if TVOS
+		[Obsolete ("Starting with tvos10.10 this API has been totally removed on tvOS.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
```

Comment:
same, no iOS Obsolete.

Reply:
There was only a message associated with the the TvOS deprecation and there doesn't seem to be more info on the web docs about the deprecation message. Should I add the same message for iOS and just say totally removed on iOS?

Reply:
Same, looking at the headers, it was completely removed.